### DISCUSSION TITLE
希望能够加入文件管理器挂载webdav网盘，并且设置同步的功能

### DISCUSSION THREAD

**Discussion by xzl01** [linuxdeepin/developer-center#3479] [Features Request & Ideas | 特性请求 & 头脑风暴]
如题，在windows上有微软的onedrive mac上有云上贵州，linux上应该也有自己的网盘系统，建议能做好webdav的网盘集成

**Comment by BLumia** [comment]
文管不清楚有没有明确支持 webdav，但终端 `gio mount davs://webdav地址` 应该就能挂上了，并且挂载后文管里应该也看得到。

**Reply by xzl01** [reply]
是，但是应该能更简单，用图形化的方式进行操作，并且支持指定文件夹进行定时备份

**Reply by BLumia** [reply]
我没有 webdav 服务端可供测试，不过印象里我之前是给文管加过网络挂载的完整支持的（尽管没完整测，当时只按 sftp 测的。其实 samba 之类走的也是完全同一套接口，都是 gvfs 的网络挂载）。可以文管里 <kbd>Ctrl + L</kbd> 在地址栏粘贴 webdav 的 URL 然后回车，看能否进入正常的挂载过程（询问凭据后应该可以挂上）
